luminoth/datasets/object_detection_dataset_test.py

```diff
@@ -0,0 +1,90 @@
+import tensorflow as tf
+import numpy as np
+
+from easydict import EasyDict
+
+from luminoth.datasets.object_detection_dataset import ObjectDetectionDataset
+
+
+class ObjectDetectionDatasetTest(tf.test.TestCase):
+    def setUp(self):
+        self.base_config = EasyDict({
+            'dataset': {
+                'dir': '',
+                'split': 'train',
+                'image_preprocessing': {
+                    'min_size': 600,
+                    'max_size': 1024,
+                },
+                'data_augmentation': {},
+            },
+            'train': {
+                'num_epochs': 1,
+                'batch_size': 1,
+                'random_shuffle': False,
+            }
+        })
+
+    def _run_augment(self, augment_config):
+        self.base_config['dataset']['data_augmentation'] = augment_config
+
+        bboxes = tf.placeholder(tf.int32, shape=self.bboxes.shape)
+        image = tf.placeholder(tf.int32, shape=self.image.shape)
+
+        model = ObjectDetectionDataset(self.base_config)
+        image, bboxes, applied_data_augmentation = model._augment(
+            image, bboxes)
+
+        with self.test_session() as sess:
+            image, bboxes, applied_data_augmentation = sess.run(
+                [image, bboxes, applied_data_augmentation], feed_dict={
+                    bboxes: self.bboxes,
+                    image: self.image,
+                })
+            return image, bboxes, applied_data_augmentation
+
+    def testSortedAugmentation(self):
+        """
+        Tests that the augmentation is applied in order
+        """
+        self.image = np.random.randint(low=0, high=255, size=(600, 800, 3))
```

Comment:
Don't use `self\.` as a shortcut to not passing parameters as arguments, `image` and `bboxes` are not object attributes.